.eslintrc.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    'no-proto': 0,
   },
-  plugins: ['jest']
+  plugins: ['jest'],
```

Comment:
Consider placing the 'plugins' property before the 'rules' property for better readability and to follow conventional ESLint configuration order.